include/ck/tensor_operation/gpu/element/element_wise_operation.hpp

```diff
@@ -172,6 +172,42 @@ struct AddAdd
     }
 };
 
+// C = A * B
+// E = (C + D0) x D1
+struct AddMultiply
+{
+    template <typename E, typename C, typename D0, typename D1>
+    __host__ __device__ void operator()(E& e, const C& c, const D0& d0, const D1& d1) const;
+
+    template <>
+    __host__ __device__ void operator()<half_t, half_t, half_t, half_t>(half_t& e,
+                                                                        const half_t& c,
+                                                                        const half_t& d0,
+                                                                        const half_t& d1) const
+    {
+        const half_t y = c + d0 + d1;
```

Comment:
Shouldn't you multiply by `d1`  as in operation comment and name? 
```suggestion
        const half_t y = (c + d0) * d1;
```